idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/solid_phase_thermo.py

```diff
@@ -666,8 +669,7 @@ def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
 
         if self.is_property_constructed("sum_component_eqn"):
-            for t, v in self.sum_component_eqn.items():
-                iscale.constraint_scaling_transform(
-                    v,
-                    iscale.get_scaling_factor(self.mass_frac_comp[t]),
-                    overwrite=False)
+            iscale.constraint_scaling_transform(
+                self.sum_component_eqn,
+                iscale.get_scaling_factor(self.mole_frac_comp['CH4']),
```

Comment:
Select a component from the solids component list. CH4 is only in the gas property package.